enhancements/windows-containers/disconnected-support.md

```diff
@@ -0,0 +1,374 @@
+This workflow will be used to run the existing WMCO e2e test suite to ensure that WMCO runs as expected in disconnected 
+environments with no regression. We will add a few test cases to explicitly check the state of registry settings on Windows
+nodes.
+QE should cover disconnected environments on all supported platforms when validating this feature.
+
+### User Stories
+
+Story 1: Set `config_path` in containerd_conf.toml
+Story 2: WMCO generates hosts.toml files
+Story 3: WMCO copies over generated hosts.toml files to each Windows instance
+
+User stories can also be found within the JIRA epic: [WINC-936](https://issues.redhat.com/browse/WINC-936)
+
+### Workflow Description and Variations
+
+**cluster creator** is a human user responsible for deploying a cluster.
+**cluster administrator** is a human user responsible for managing cluster settings including network egress policies.
+
+There are 4 different workflows that affect the disconnected use case.
+1. A cluster creator brings up a new disconnected cluster and configures mirror registry settings at install time
+2. A cluster administrator introduces new mirror registry settings or changes exisiting ones during runtime
+   This would occur through the creation, deletion, or update IDMS/ITMS resources.
+3. A cluster administrator transitions a connected cluster *with existing mirror registry settings* to a disconnected cluster
+4. A cluster administrator transitions a connected cluster *without existing mirror registry settings* to a disconnected cluster
+
+
+In cases 1, 2 & 3, Windows nodes can be joined to the cluster after altering registry settings,
+which would result in WMCO applying mirror settings during initial node configuration. 
+
+In case 2, for any existing Windows nodes, WMCO will react by updating each instance's state to consume the new registry config.
+
+In case 3, existing Windows nodes will not be re-configured. Existing registry settings will stay applied.
+
+In case 4, WMCO will take no action as there are no settings to configure. Windows nodes will not be able to pull images
+until mirror registry settings are configured by the administrator.
+
+### Risks and Mitigations
+
+If node cleanup fails, instances may retain their containerd configuration. This could be a concern for BYOH instances
+particularly. An event should be generated to alert cluster admins manual cleanup is required to remove WMCO-created files.
+
+Another concern is if a user's mirror configuration has overlapping settings/clashes. As on the Linux side, it is upon
+the user to ensure there is no clash between mirrors across their IDMS and ITMS resources.
+
+<!--
+Containerd may not support all the fields available in IDMS/ITMS resources.
+Unsupported fields will be ignored and will not impact containerd registry config/default behavior.
+-->
+
+### Drawbacks
+
+The only drawbacks are the increased complexity of WMCO and the potential complexity of debugging customer cases that
```

Comment:
You need to call out the scaling issues due to the sequential nature of copying the files on IDMS/ITMS change.